Source/WebCore/dom/TreeScope.cpp

```diff
@@ -572,4 +577,165 @@ ExceptionOr<void> TreeScope::setAdoptedStyleSheets(Vector<RefPtr<CSSStyleSheet>>
     return ensureAdoptedStyleSheets().setSheets(WTFMove(sheets));
 }
 
+struct SVGResouresMap {
+    WTF_MAKE_NONCOPYABLE(SVGResouresMap); WTF_MAKE_FAST_ALLOCATED;
```

Comment:
Should use `WTF_MAKE_STRUCT_NONCOPYABLE` and then we won’t need to write `public` below.

Reply:
oh, didn't know we had this. Fixed.